app/src/main/java/org/mozilla/reference/browser/browser/BrowserFragment.kt

```diff
@@ -63,23 +63,26 @@ class BrowserFragment : BaseBrowserFragment(), UserInteractionHandler {
                 freshTab,
                 fresh_tab_toolbar,
                 engineView,
-                requireComponents.core.sessionManager)
-                .addNewsFeature(
-                    newsView,
-                    lifecycleScope,
-                    requireComponents.useCases.sessionUseCases.loadUrl,
-                    requireComponents.useCases.getNewsUseCase,
-                    requireComponents.core.icons
-                ).addTopSitesFeature(
-                    topSitesView,
-                    requireComponents.useCases.sessionUseCases.loadUrl,
-                    requireComponents.useCases.historyUseCases.getTopSites,
-                    requireComponents.core.icons
-                ),
-                owner = this,
-                view = view
+                requireComponents.core.sessionManager,
+                sessionId)
+            .addNewsFeature(
```

Comment:
I would suggest to not change the indentation when introducing functional changes, otherwise git blame becomes less useful.